src/wasm-lib/grackle/src/native_functions/sketch/stdlib_functions.rs

```diff
@@ -204,7 +375,7 @@ impl Callable for LineTo {
             // Then its `relative` field.
             Instruction::SetPrimitive {
                 address: start_of_line + 1 + length_of_3d_point,
-                value: false.into(),
+                value: (opts.at == At::RelativeX || opts.at == At::RelativeY || opts.at == At::RelativeXY).into(),
```

Comment:
could be a method `is_relative` on the `At` type instead, and use a `match` so that if we add more variants, we'll be forced to handle them.

Reply:
Ah yeah, great idea